src/extensions/default/bramble/lib/RemoteEvents.js

```diff
@@ -125,14 +125,22 @@ define(function (require, exports, module) {
                 wordWrap: PreferencesManager.get("wordWrap")
             });
         });
-        
+
         // Listen for changes to allow javascript
         PreferencesManager.on("change", "allowJavaScript", function () {
             sendEvent({
                 type: "bramble:allowJavaScriptChange",
                 allowJavaScript: PreferencesManager.get("allowJavaScript")
             });
         });
+        // Listen for changes to allow javascript
+        var Lint = PreferencesManager.getExtensionPrefs("interactive-linter");
+        Lint.on("change", "allowLint", function () {
```

Comment:
This is wrong.  Check out https://github.com/mozilla/brackets/pull/702/files which is pretty similar to what you're doing.